hack/tools/requirements.env

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env bash
 # Copyright 2020 The Kubernetes Authors.
 # SPDX-License-Identifier: Apache-2.0
 
-set -o nounset
+# after updating any of these version, please run 'make reconcile-requirements' from the root Makefile
```

Comment:
"after updating any of these version, please run 'make reconcile-requirements' at the root project directory"

as an option?

Reply:
sound great, fixed